### ISSUE TITLE
torchvision-0.12.0: Unable to find torch_shm_manager

### ISSUE THREAD

**Issue by equaeghe** [aclex/pytorch-ebuild#17]
When emerging torchvision-0.12.0, I get the following error:

```
RuntimeError: Unable to find torch_shm_manager at /usr/lib/python3.9/site-packages/torch/bin/torch_shm_manager
 * ERROR: sci-libs/torchvision-0.12.0::aclex-pytorch failed (compile phase)
```

Indeed, `torch_shm_manager` is not at that location:

```
# updatedb
# locate torch_shm_manager
/usr/bin/torch_shm_manager
# equery b torch_shm_manager
 * Searching for torch_shm_manager ... 
sci-libs/pytorch-1.11.0 (/usr/bin/torch_shm_manager)
```

I do not know whether this is an issue on my side or with one of the ebuilds from this repository.

**Comment by aclex** [comment]
Thanks for the report! Hmm, that's obviously a problem in sci-libs/pytorch-1.11.0, which I've also happened to update recently. I thought it's already fixed, I would double check it then.

**Comment by aclex** [comment]
@equaeghe could you please also write USE flags you use to build sci-libs/pytorch? There seems to be different install settings depending on them, I added a condition to the ebuild for that, and probably there's something wrong with it.

**Comment by equaeghe** [comment]
```
# eix -I pytorch
[I] sci-libs/pytorch [1]
     Available versions:  (~)1.5.0(0/1.5.0)*l (~)1.5.1(0/1.5.1)*l (~)1.6.0(0/1.6.0)*l (~)1.7.0(0/1.7.0)*l (~)1.7.1(0/1.7.1)*l (~)1.8.0(0/1.8.0)*l (~)1.8.1(0/1.8.1)*l (~)1.9.0(0/1.9.0)*l (~)1.10.0(0/1.10.0)*l (~)1.11.0(0/1.11.0) {asan atlas caffe2 cuda doc eigen +fbgemm ffmpeg gflags glog +gloo leveldb lmdb mkl +mkldnn mpi namedtensor +nnpack numa +numpy +observers openblas opencl opencv +openmp +python +qnnpack redis rocm static tbb test tools zeromq PYTHON_TARGETS="python3_8 python3_9 python3_10"}
     Installed versions:  1.11.0(0/1.11.0)(11:31:26 PM 04/18/2022)(fbgemm gloo mkldnn nnpack numpy observers openblas openmp python qnnpack -asan -atlas -caffe2 -cuda -doc -eigen -ffmpeg -gflags -glog -leveldb -lmdb -mkl -mpi -namedtensor -numa -opencl -opencv -redis -rocm -static -tbb -test -tools -zeromq PYTHON_TARGETS="python3_9 -python3_8 -python3_10")
     Homepage:            https://pytorch.org/
     Description:         An open source machine learning framework

[1] "aclex-pytorch" /var/db/repos/aclex-pytorch
```

So `fbgemm gloo mkldnn nnpack numpy observers openblas openmp python qnnpack` are active.

**Comment by aclex** [comment]
Thank you very much! I seem to find the issue, will push the update now, hopefully it would fix it for you as well.
